test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -34,6 +37,7 @@ def func(tensor):
         tensor = common_utils.get_whitenoise()
         self._assert_consistency(func, tensor)
 
+    @skipCUDAIfRocm
```

Comment:
If you see what function is not supported by ROCm in the error message, could you add a comment?

Reply:
This also gives a a mismatch error.
```
==================================================================================== FAILURES ====================================================================================
_____________________________________________________________________ TestFunctionalFloat32.test_griffinlim ______________________________________________________________________

self = <torchaudio_unittest.functional.torchscript_consistency_cuda_test.TestFunctionalFloat32 testMethod=test_griffinlim>

    @skipCUDAIfRocm
    def test_griffinlim(self):
        def func(tensor):
            n_fft = 400
            ws = 400
            hop = 200
            window = torch.hann_window(ws, device=tensor.device, dtype=tensor.dtype)
            power = 2.
            momentum = 0.99
            n_iter = 32
            length = 1000
            rand_int = False
            return F.griffinlim(tensor, window, n_fft, hop, ws, power, n_iter, momentum, length, rand_int)
    
        tensor = torch.rand((1, 201, 6))
>       self._assert_consistency(func, tensor)

test/torchaudio_unittest/functional/torchscript_consistency_impl.py:55: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test/torchaudio_unittest/functional/torchscript_consistency_impl.py:24: in _assert_consistency
    self.assertEqual(ts_output, output)
/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/common_utils.py:1251: in assertEqual
    super().assertTrue(result, msg=self._get_assert_msg(msg, debug_msg=debug_msg))
E   AssertionError: False is not true : Tensors failed to compare as equal!With rtol=1.3e-06 and atol=1e-05, found 996 element(s) (out of 1000) whose difference(s) exceeded the margin of error (including 0 nan comparisons). The greatest difference was 0.20305343717336655 (-0.11751212924718857 vs. 0.08554130792617798), which occurred at index (0, 53).
================================================================================ warnings summary ================================================================================
test/torchaudio_unittest/functional/torchscript_consistency_cuda_test.py::TestFunctionalFloat32::test_griffinlim
  /opt/conda/lib/python3.6/site-packages/torch/functional.py:656: UserWarning: istft will require a complex-valued input tensor in a future PyTorch release. Matching the output from stft with return_complex=True.  (Triggered internally at  /var/lib/jenkins/pytorch/aten/src/ATen/native/SpectralOps.cpp:793.)
    normalized, onesided, length, return_complex)

-- Docs: https://docs.pytest.org/en/stable/warnings.html
============================================================================ short test summary info =============================================================================
FAILED test/torchaudio_unittest/functional/torchscript_consistency_cuda_test.py::TestFunctionalFloat32::test_griffinlim - AssertionError: False is not true : Tensors failed to...
```